plugins/available/nvm.plugin.bash

```diff
@@ -9,13 +9,17 @@ about-plugin 'node version manager configuration'
 
 export NVM_DIR=${NVM_DIR:-$HOME/.nvm}
 # This loads nvm
-if command -v brew &>/dev/null && [ -s $(brew --prefix nvm)/nvm.sh ]
+if [ -s "$NVM_DIR/nvm.sh" ]
 then
-  . $(brew --prefix nvm)/nvm.sh
+  source "$NVM_DIR/nvm.sh"
 else
-  [ -s "$NVM_DIR/nvm.sh" ] && . "$NVM_DIR/nvm.sh"
+  if command -v brew &>/dev/null && [ -s $(brew --prefix nvm)/nvm.sh ]
+  then
+    source $(brew --prefix nvm)/nvm.sh
```

Comment:
We're running `brew --prefix nvm` twice here, once in the `if` and then again when `sourcing` - we might want to combine into a single invocation using a variable. Running `brew --prefix` is costly, the overhead of using a variable will still be worth the potential savings.